app/(gcforms)/[locale]/(form administration)/form-builder/[id]/settings/api/components/ApiKeyButton.tsx

```diff
@@ -5,20 +5,20 @@ import { useTranslation } from "@i18n/client";
 import { EventKeys, useCustomEvent } from "@lib/hooks/useCustomEvent";
 import { DeleteKeyButton } from "./DeleteKeyButton";
 import { SubmitButton as GenerateApiKeyButton } from "@clientComponents/globals/Buttons/SubmitButton";
+import { useFormBuilderConfig } from "@lib/hooks/useFormBuilderConfig";
 
 type ApiKeyButtonProps = {
-  keyId?: string | false;
   showDelete?: boolean;
   i18nKey?: string;
 };
 
 export const ApiKeyButton = ({
-  keyId,
   showDelete = false,
   i18nKey = "settings.api.generateKey",
 }: ApiKeyButtonProps) => {
   const { t } = useTranslation("form-builder");
   const { id } = useParams();
+  const { apiKeyId: keyId } = useFormBuilderConfig();
```

Comment:
Can just use apiKeyId here --- we don't need to alias

Reply:
When I could, I left variable names as is to avoid changing extra code, why the aliases. Do you find it hard to read?